### ISSUE TITLE
makefile error

### ISSUE THREAD

**Issue by zking-z** [amd/xdna-driver#306]
Hello, currently I am using XDNA2 (Strix Point). If I want to use the "Introduction to Scalar and Vector Programming" in the AI Engine, as described on the website [https://docs.amd.com/r/en-US/ug1079-ai-engine-kernel-coding/AI-Engine-API], do I need to install the XDNA™ Driver? Also, do I need to install XRT? Currently, I am using Ubuntu 22.04.3 with kernel version 6.8.0-48-generic. When building the kernel and running make -j \nproc` bindeb-pkg`, I encounter the following error:
make[4]: *** [Makefile:1934: .] Error 2
make[3]: *** [debian/rules:74: build-arch] Error 2
dpkg-buildpackage: error: make -f debian/rules binary subprocess returned exit status 2
make[2]: *** [scripts/Makefile.package:121: bindeb-pkg] Error 2
make[1]: *** [/home/never/linux/Makefile:1555: bindeb-pkg] Error 2
make: *** [Makefile:240: __sub-make] Error 2
How should I address this issue?

**Comment by houlz0507** [comment]
No. you do not need XRT or XDNA driver. Maybe you can get the right support from the same document. :) Close for now.
